tests/CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ addUnitTest(test_precision)
 addUnitTest(test_general)
 addUnitTest(test_random)
 addUnitTest(test_stp_bv_performance)
+addUnitTest(test_kanehiro)
```

Comment:
This file is not in the PR. Also there is probably a more descriptive name to find :)